tests/integration/security/test_xss.py

```diff
@@ -42,9 +42,10 @@ def test_xss001_banned_protocols(dash_duo):
     ):
 
         element = dash_duo.find_element(element_id)
+        prop_value = element.get_attribute(prop)
         assert (
-            element.get_attribute(prop) == "about:blank"
-        ), f"Failed prop: {element_id}.{prop}"
+            prop_value == "about:blank"
```

Comment:
If we are inlining the sanitization logic to our codebase, probably makes sense to add a few more tests along the lines of [these](https://github.com/braintree/sanitize-url/blob/0d5140d7f7489f46472a545635be99f8282f83e4/src/__tests__/index.test.ts#L102)

Reply:
I added more tests cases, some of the sanitization in that was done in the lib were not valid as dash prop.